pepper-apis/scripts/init-gae-egress-proxy-vm.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+#
+# Creates a VM to act as a proxy for GAE outbound traffic.
+
+NAME=$(basename "$0")
+if (( $# < 1 )); then
+  echo "usage: $NAME <PROJECT_ID>"
+  exit 1
+fi
+
+PROJECT_ID="$1"
+ENV_NAME="${PROJECT_ID/broad-ddp-/}"
+INSTANCE_NAME="gae-egress-proxy-${ENV_NAME}101"
+
+echo "using gcp project: $PROJECT_ID"
+echo "using instance name: $INSTANCE_NAME"
+
+echo ""
+echo "=> reserving static external ip..."
+gcloud --project="$PROJECT_ID" compute addresses create "$INSTANCE_NAME" --region=us-central1
+gcloud --project="$PROJECT_ID" compute addresses describe "$INSTANCE_NAME" --region=us-central1
+
+echo ""
+echo "=> creating vm instance..."
+gcloud --project="$PROJECT_ID" compute instances create "$INSTANCE_NAME" \
+  --zone=us-central1-a \
+  --machine-type=g1-small \
+  --image-project=ubuntu-os-cloud \
+  --image-family=ubuntu-minimal-1604-lts \
```

Comment:
I noticed that all the other VMs that we have uses 1604, so I just went with that.

Reply:
Unless there are specific compatibility concerns we are addressing, there's no reason we shouldn't be using a more recent LTS release.

Reply:
I don't think we have used 1804 or 2004 images in our VMs before, so I'm not sure about compatibility issues. I bumped this up to 1804 bionic since it looks like that's what we're sourcing the docker images from.